src/cli/java/org/commcare/util/engine/CommCareConfigEngine.java

```diff
@@ -214,9 +214,11 @@ private void init(String profileRef) {
         }
     }
 
-    public void installAppFromReference(String profileReference) throws UnresolvedResourceException,
+    public void installAppFromReference(String profileReference)
+            throws UnresolvedResourceException,
             UnfullfilledRequirementsException, InstallCancelledException {
-        ResourceManager.installAppResources(platform, profileReference, this.table, true);
+        ResourceManager.installAppResources(platform, profileReference, this.table, true,
+                Resource.RESOURCE_AUTHORITY_LOCAL);
```

Comment:
Yea I initially thought that this was going to be more complicated too, but I traced the code path and it seemed like it's only ever offline install